deployment-templates/deploy-explain.yaml.sample

```diff
@@ -0,0 +1,90 @@
+# Copyright 2017 Google Inc.
```

Comment:
Can you move this to the newly created samples/ directory? You can also drop the .sample extension as that would be redundant.